### DISCUSSION TITLE
Calibration

### DISCUSSION THREAD

**Discussion by J-Pipe** [emporia-vue-local/esphome#251] [Q&A]
Dumb question, regarding the voltage calibration,
```

<in-use calibration value> * <accurate voltage> / <reporting voltage>

```

How do I get the "accurate" voltage? I assume take a multimeter reading of the voltage between phase X and ground?
Is this what everyone is doing?

**Comment by daelsc** [comment]
correct.  use a high quality digital multi meter to measure voltage between each phase and neutral.
